### PR TITLE
Add docker_version variable support

### PR DESCRIPTION
**Pull request by mum0l**
Tested in AWS on Ubuntu, RedHat and deployment works properly.
Docker version for RedHat adjusted as per version available in our repo.
QUESTION: Should I test it on Azure and on Baremetal also in both version Ubuntu/Redhat?

### CODE REVIEW

**Comment by mkyc** [comment]
@mum0l Azure test is also important.

**Comment by mum0l** [comment]
@mkyc sure, I will learn Azure DevOPS this weekend, and try to run tests in Monday

**Comment by mkyc** [comment]
You do not have to learn Azure DevOps now. Just run it in Azure Cloud setup. Maybe @przemyslavic can help you with that?

**Comment by przemyslavic** [comment]
@mkyc, @mum0l sure, I'll test all supported configurations.

File: core/src/epicli/data/common/ansible/playbooks/roles/docker/defaults/main.yml [hunk 1]
```diff
@@ -4,4 +4,8 @@ docker_logging:
     max_file_size: 10m # The maximum size of the log before it is rolled. A positive integer plus a modifier representing the unit of measure (k, m, or g)
     max_files: 2       # The maximum number of log files that can be present
 
-docker_version: 18.09.6
\ No newline at end of file
+docker_version: "5:18.09.*"
+docker_cli_version: "5:18.09.*"
+
+docker_version_rh: "3:18.09.*"
+docker_cli_version_rh: "1:19.03.*"
```
Comment (to-bar) [inline core/src/epicli/data/common/ansible/playbooks/roles/docker/defaults/main.yml:11]:
```suggestion
docker_version:
  Debian: "5:18.09.*"
  RedHat: "18.09.*"
```
We would like to have the same version for `docker-ce` and `docker-ce-cli`.

**Comment by przemyslavic** [comment]
@mum0l also remember about requirements.txt files (if you change package versions)

**Comment by mum0l** [comment]
Thanks for comments guys! I will prepare new version, and put a comment here.

**Comment by mum0l** [comment]
Yesterday I did final fix, so now everything should be good.

@mkyc thx for approval.
@przemyslavic - today I test deployment via Azure Devops, building a cluster in both AWS and Azure, and no issues were found. Please review and approve.

**Comment by mum0l** [comment]
Okey, its done. 
1. Two tasks slimmed into one.
2. Adjusted variables
3. Two useless packages removed: **apt-transport-https** and **ca-certificates** (as they are installed thru: core\src\epicli\data\common\ansible\playbooks\roles\common\tasks\Debian.yml).

Tested in Azure Pipeline and deployment works properly on Debian/RH both on AWS and Azure.

**Pull request merged**